poetry.lock

```diff
@@ -1,105 +1,121 @@
-# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
```

Comment:
I don't think it's a problem, but you may want to run a `poetry self update` (https://python-poetry.org/docs/cli/#self-update) to update your local `poetry`.